src/test/regress/sql/multi_unsupported_worker_operations.sql

```diff
@@ -204,6 +204,12 @@ DELETE FROM pg_dist_shard_placement WHERE nodeport = :worker_2_port AND shardid
 -- master_get_new_placementid
 SELECT master_get_new_placementid();
 
+-- Show that sequences can be created and dropped on worker nodes
+CREATE TABLE some_table_with_sequence(a int, b BIGSERIAL, c BIGSERIAL);
+DROP TABLE some_table_with_sequence;
+CREATE SEQUENCE some_sequence;
+DROP SEQUENCE some_sequence;
```

Comment:
It will just DROP the default from the owner column of the MX table (and shards) but I think this is the expected behavior as we discussed in #1181, isn't it?